src/components/Methodology.js

```diff
@@ -107,58 +81,41 @@ const Methodology = props => {
 
           <h4>City Ranking</h4>
-          <p>Cities are ranked based on the map error rate which is calculated as below.</p>
-          <p>For atlas features within the subject area:</p>
-          <img className="equation" src={require('../images/formula_simple.png')} alt="equation for calculating city ranking" />
-          <p>The lower error rate a city has, the higher it is ranked. The error rate is normalized so that more weights are 
-            assigned to atlas features that are flagged multiple times.</p>
+          <p>Cities are ranked based on the map error rate with error OSM features over all OSM features. The lower error rate a city has, the higher it is ranked.</p>
+
+          <h4>OSM PBF Extraction</h4>
+          <p> The OSM data used was extracted from a full OSM history PBF file (<a href="https://planet.openstreetmap.org/">Source</a>). The goal of the initial phase of data processing was to have two extracts for each city, one containing 2018 data and the other 2019 data. This was achieved using a tool called <a href="https://osmcode.org/osmium-tool/">Osmium</a>.</p>
+          <p> The first step was to cut the full planet OSM history PBF into a history file for each city using Osmium's 'extract' tool. The tool used geojson boundaries of each city to perform a soft clip for each, generating the new city history PBFs.</p>
+          <p>Next, each of the city history files were time sliced. Using Osmium's 'time-filter' tool each history file was filtered so that the data appeared the same as it would have at a specific point in time in OSM. For each city this was done twice, creating a PBF for April 7th at 10am UTC for both 2018 and 2019. At this point, the new time slice PBFs were in standard OSM PBF format, as opposed to OSM history files.</p>
+          <p>Finally, extraneous Nodes were cleaned from the time slice PBFs using a simple python script. The Nodes in question did not have an spatial information, as they were remnants of features that had been deleted at that point in time.</p>
+
+          <h4>MQM Extent Enhancement</h4>
+          <p>An addition to MQM was incorporated into our methodology this year, to better compare differences over time. The additional feature allows the extent of MQM results to be locked to a specified bounding box, as opposed to being dynamically set by the extent of the input data. This allowed the results of two MQM runs to share the same extent despite small variances in the spatial bound of the input data.</p>
+
+          <h4>Social-Economic Score in MQM</h4>
+          <p> To incorporate the social-economic metrics, we collect data from the U.S. Census Bureau's American Community Survey. This data source includes different demographic information which we used as a proxy for road usage in each city based on each census tract.</p> 
```

Comment:
To reprioritize map error hot-spots by usage, we collect data from the U.S. Census Bureau's American Community Survey. In particular, we used population data from 'Selected Characteristics of the Total and Native Populations in the United States' and 'no vehicle available' data from 'Means of Transportation to Work by Selected Characteristics' as estimates of road usage level in each city. Both are 2017 ACS 5-year estimates on the census tract level.</p> 
	          <p> We generated the MQM grids for population and car ownership through the following steps: first, joined the census data to the city census tracts. Then, we generated raster layers for each data category. Afterwards, we generated the MQM grids using the city boundary, and calculated the mean value of the raster layer in each grid using the zonal statistics tool in QGIS. Finally, we normalized the values into a scale of 0 - 100% and reclassify them into five levels by quantile. </p>